server/service/integration_mdm_test.go

```diff
@@ -7119,7 +7119,7 @@ func (s *integrationMDMTestSuite) TestMDMEnabledAndConfigured() {
 			Token:  uuid.New().String(),
 			Bytes:  []byte("foo"),
 			Sha256: []byte("foo-sha256"),
-		}))
+		}, nil))
```

Comment:
sounds great, thank you! if it can be used as inspiration, we have integration tests that use s3 (via minio) for the good old sandbox https://github.com/fleetdm/fleet/blob/1a0fe077e89a2d48ba907d0e8eb175bd7f438a33/server/service/integration_sandbox_test.go#L36-L45